### PR TITLE
YONK-16: Added split module store for tests

### PR DESCRIPTION
**Pull request by aamishbaloch**

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: progress/tests.py [hunk 1]
```diff
@@ -71,25 +73,25 @@ def _create_course(self, start=None, end=None):
         self.chapter1 = ItemFactory.create(
             category="chapter",
             parent_location=self.course.location,
-            data=test_data,
+            # data=test_data,
```
Comment (ziafazal) [inline progress/tests.py]: commented code

File: progress/tests.py [hunk 2]
```diff
@@ -392,7 +394,10 @@ def test_progress_calc_on_vertical_with_detached_module(self):
         self.assertEqual(len(progress), 0)
 
     def test_receiver_on_course_deleted(self):
-        self._create_course(start=datetime(2010, 1, 1, tzinfo=UTC()), end=datetime(2020, 1, 1, tzinfo=UTC()))
+        self._create_course(
```
Comment (ziafazal) [inline progress/tests.py]: We should add dual module_store support for all those tests where we are creating courses with `self._create_course`

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch Could you please incorporate my suggestion?

**Comment by aamishbaloch** [comment]
@ziafazal Done with the suggested changes. Can you review this?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**